### PR TITLE
Update earlier slides, split out a step0 

### PR DESCRIPTION
**Pull request by AlanSl**
- Adds "install expo" nag box to first few slides
- Makes "Start here" into a slide we can leave up on the screen, telling people to clone the workshop
- Adds a "step 0" splitting out basic Hello World, is your setup stuff from our app
- Adds slides for step 0 and step 1 with screenshots of how it should look

### CODE REVIEW

File: slides/slides.md [hunk 1]
```diff
@@ -75,8 +105,72 @@ class: dense
...
+
 ---
 
 # Start here
 
-https://reactnative.dev/docs/environment-setup
+- Clone this workshop!
+  1. https://github.com/nearform/react-native-workshop
+  2. Open these slides from the readme
+  3. Look in `./step1` etc for code and more info in `README`s
```
Comment (robwalkerco) [inline slides/slides.md:122]: Should think link to Step 0?
```suggestion
  3. Look in `./step0` etc for code and more info in `README`s
```

File: slides/slides.md [hunk 2]
```diff
@@ -75,8 +105,72 @@ class: dense
...
+https://expo.dev/client
+</div>
+  
+---
+
+# [Step 0: Hello world](https://github.com/nearform/react-native-workshop/tree/main/step0)
+
+- Create and run starter Expo app:
+  - `npx create-expo-app NodeConfGame`
+    - (want TypeScript not JS? Add `-t`)
```
Comment (robwalkerco) [inline slides/slides.md:141]: I think that flag also requires a value for the template to use

I used `npx create-expo-app -t expo-template-blank-typescript` when creating the original project.

File: step0/README.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+```sh
+npx create-expo-app NodeConfGame
+```
+
+This creates a template Expo app in JavaScript
+
+#### Want TypeScript?
+
+If you prefer to use TypeScript, add the `-t` flag to the above command.
```
Comment (robwalkerco) [inline step0/README.md:19]: See earlier comment regarding adding a template value after the -t flag

File: step0/README.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+### 5. Did it work?
+
+If everything worked, you should see a notice about building JavaScript, then a short loading splash screen, then a white screen with text like this:
+
+> Open up App.js to start working on your app!
+
+Open `App.js`, make some small change like editing this text, save, and it should automatically update the app.
+
+#### Troubleshooting
```
Comment (robwalkerco) [inline step0/README.md:63]: Another thing to perhaps add here is that if app is not loading, to try hitting `r` in the Expo bundler to reload the app in Expo Go.

**Review by robwalkerco — APPROVED** [review]
A few small suggestions, but it's looking good otherwise.

**Comment by AlanSl** [comment]
Thanks for the review, I've done the changes but gone with `--template` without hardcoding a template name because it gives an in-console menu of templates which might be useful for beginners to see

**Pull request merged**